Training2/src/src/task1/index.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { useState } from "react";
+import Modal from "./Modal";
+
+const Greeting = () => {
+  const [isShowing, setIsShowing] = useState(false);
+
+  const handleClick = () => {
+    setIsShowing(!isShowing);
```

Comment:
@doankhanh222 We should not use !isShowing, we should use setIsShowing with the callback for every setState using previous state to calculate the new state value